vendor/github.com/blang/semver/range.go

```diff
@@ -327,7 +327,7 @@ func expandWildcardVersion(parts [][]string) ([][]string, error) {
 	for _, p := range parts {
 		var newParts []string
 		for _, ap := range p {
-			if strings.Contains(ap, "x") {
+			if strings.Index(ap, "x") != -1 {
```

Comment:
It seems there are two or more libraries being updated here, not just adding/removing files we didn't use.

Reply:
ping @spangenberg